produce/lib/produce/options.rb

```diff
@@ -137,7 +137,19 @@ def self.available_options
                                      default_value: CredentialsManager::AppfileConfig.try_fetch_value(:itc_team_name),
                                      verify_block: proc do |value|
                                        ENV["FASTLANE_ITC_TEAM_NAME"] = value.to_s
-                                     end)
+                                     end),
+        FastlaneCore::ConfigItem.new(key: :merchant_name,
+                                      short_option: "-r",
+                                      env_name: "PRODUCE_MERCHANT_NAME",
+                                      description: "Name for the merchant that is created",
+                                      optional: true),
+        FastlaneCore::ConfigItem.new(key: :merchant_identifier,
+                                       short_option: "-o",
+                                       env_name: "PRODUCE_MERCHANT_IDENTIFIER",
+                                       description: "Merchant identifier for the merchant",
+                                       verify_block: proc do |value|
+                                         UI.user_error!("Merchant ID identifiers should start with 'merchant.'") unless value.start_with?("merchant.")
```

Comment:
Can we put in the description that the value should start with `merchant.` or, maybe we can be smart about it and prepend to the string if it doesn't already start with that string?